package io.io.Exception;

public class ExistingUsernameException extends Exception{
}
